sdk/identity/identity/src/credentials/browserCustomizedOptions.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+
+/**
+ * Shared configuration options for browser customization
+ */
+export interface BrowserCustomizedOptions {
```

Comment:
Not sure I love this interface/property name but I'm struggling with what it should be called.

browserRedirectOptions?

Reply:
The other languages have already shipped this API last GA. I'll probably go with whatever they decided across which is `BrowserCustomization`